test/ops/prelu.js

```diff
@@ -0,0 +1,139 @@
+    const graph = await builder.build({y});
+    const inputs = {
+      'x': new Float32Array([
+        1, -1, 2,
+        -2, 3, -3,
+      ]),
+      'slope': new Float32Array([
+        0.1, -0.1, 0.25,
+        -0.25, -0.5, 0.5,
+      ]),
+    };
+    const outputs = {'y': new Float32Array(utils.sizeOfShape([1, 2, 3]))};
+    const result = await context.compute(graph, inputs, outputs);
+    const expected = [
+      1, 0.1, 2,
+      0.5, 3, -1.5,
+    ];
+    utils.checkValue(result.outputs.y, expected);
+  });
+
+  it('prelu 3d constant slope', async () => {
+    const builder = new MLGraphBuilder(context);
+    const x = builder.input('x', {type: 'float32', dimensions: [1, 2, 3]});
+    const slope = builder.constant(
+        {type: 'float32', dimensions: [1, 2, 3]},
+        new Float32Array([
+          0.1, -0.1, 0.25,
+          -0.25, -0.5, 0.5,
+        ]),
+    );
+    const y = builder.prelu(x, slope);
+    const graph = await builder.build({y});
+    const inputs = {
+      'x': new Float32Array([
+        1, -1, 2,
+        -2, 3, -3,
+      ]),
+    };
+    const outputs = {'y': new Float32Array(utils.sizeOfShape([1, 2, 3]))};
+    const result = await context.compute(graph, inputs, outputs);
+    const expected = [
+      1, 0.1, 2,
+      0.5, 3, -1.5,
+    ];
+    utils.checkValue(result.outputs.y, expected);
+  });
+
+  it('prelu broadcast 3d x 1d', async () => {
+    const builder = new MLGraphBuilder(context);
+    const x = builder.input('x', {type: 'float32', dimensions: [1, 2, 3]});
+    const slope = builder.constant(
+        {type: 'float32', dimensions: [1]},
+        new Float32Array([0.1]),
+    );
+    const y = builder.prelu(x, slope);
+    const graph = await builder.build({y});
+    const inputs = {
+      'x': new Float32Array([
+        1, -1, 2,
+        -2, 3, -3,
+      ]),
+    };
+    const outputs = {'y': new Float32Array(utils.sizeOfShape([1, 2, 3]))};
+    const result = await context.compute(graph, inputs, outputs);
+    const expected = [
+      1, -0.1, 2,
+      -0.2, 3, -0.3,
+    ];
+    utils.checkValue(result.outputs.y, expected);
+  });
+
+  it('prelu broadcast 3d x 2d', async () => {
+    const builder = new MLGraphBuilder(context);
+    const x = builder.input('x', {type: 'float32', dimensions: [1, 2, 3]});
+    const slope = builder.constant(
+        {type: 'float32', dimensions: [1, 3]},
+        new Float32Array([
+          0.1, -0.25, 0.5,
+        ]),
+    );
+    const y = builder.prelu(x, slope);
+    const graph = await builder.build({y});
+    const inputs = {
+      'x': new Float32Array([
+        1, -1, 2,
+        -2, 3, -3,
+      ]),
+    };
+    const outputs = {'y': new Float32Array(utils.sizeOfShape([1, 2, 3]))};
+    const result = await context.compute(graph, inputs, outputs);
+    const expected = [
+      1, 0.25, 2,
+      -0.2, 3, -1.5,
+    ];
+    utils.checkValue(result.outputs.y, expected);
+  });
+
+  it('prelu broadcast 3d x 3d', async () => {
```

Comment:
This test could fail by Wasm backend (TF.js Wasm backend), reported this issue https://github.com/tensorflow/tfjs/issues/7458 to TF.js.